pkg/logql/ast.go

```diff
@@ -456,6 +478,8 @@ func (r logRange) String() string {
 	return sb.String()
 }
 
+func (r *logRange) Shardable() bool { return r.left.Shardable() }
```

Comment:
Not sure this was a Expr.

Reply:
You're right, but it's called via `rangeAggExpr.Shardable() -> logRange.Shardable() -> LogSelectorExpr.Shardable()` so it seemed more obvious to use the same method :).